pkg/logs/message/message.go

```diff
@@ -65,23 +83,25 @@ func NewMessageFromLambda(content []byte, origin *Origin, status string, utcTime
 	return &Message{
 		Content:            content,
 		Origin:             origin,
-		status:             status,
+		Status:             status,
 		IngestionTimestamp: ingestionTimestamp,
-		Timestamp:          utcTime,
-		Lambda: &Lambda{
-			ARN:       ARN,
-			RequestID: reqID,
+		ServerlessExtra: ServerlessExtra{
```

Comment:
👌 This is so much better. Will make following and tracing this codepath and the usage of these properties much easier.